### PR TITLE
[60921771] Add usc-chan project

### PR DESCRIPTION
**Pull request by emerencia**
Fixes https://freedcamp.com/view/3448368/tasks/panel/task/60921771

### CODE REVIEW

File: projects/usc-chan/seeds/questionnaires/usc_chan.rb [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    maxlength: 4,
+    placeholder: '1234',
+    min: 0,
+    max: 9999,
+    required: true
+  }, {
+    id: :v10,
+    type: :textfield,
+    placeholder: 'This is default text',
+    title: 'Example of a small free text field'
```
Comment (pmkihara) [inline projects/usc-chan/seeds/questionnaires/usc_chan.rb:81]: I was wondering what this small free text field was and what was the difference to the others, but it seems it's the same as :v6

File: projects/usc-chan/seeds/seeds.rb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                   mobile_phone: phone,
+                   role: student_hash[:role],
+                   locale: 'en')
+  end
+  puts 'Generating people - Finished'
+
+  # Solo
+  puts ''
+  protocol = Protocol.find_by(name: solo_protocol)
+  person = Team.find_by(name: usc_team).roles.where(group: Person::SOLO).first.people[0]
```
Comment (pmkihara) [inline projects/usc-chan/seeds/seeds.rb:38]: Isn't `where(group: Person::SOLO).first` the same as `find_by(group: Person::SOLO)`?

File: projects/usc-chan/seeds/seeds.rb [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                           future: TimeTools.increase_by_duration(prot_start, -1.second))
+  responseobj = prot_sub.responses.first
+  invitation_set = InvitationSet.create!(person: person)
+  responseobj.update!(open_from: 1.minute.ago, invitation_set: invitation_set)
+  invitation_token = invitation_set.invitation_tokens.create!
+  puts "USC questionnaire: #{invitation_set.invitation_url(invitation_token.token_plain)}"
+
+  # USC informed consent
+  protocol = Protocol.find_by(name: solo_protocol)
+  person = Team.find_by(name: usc_team).roles.where(group: Person::SOLO).first.people[1]
```
Comment (pmkihara) [inline projects/usc-chan/seeds/seeds.rb:57]: Also here, could it `where(group: Person::SOLO).first` be replaced by `find_by(group: Person::SOLO)`?

**Review by pmkihara — APPROVED** [review]
LGTM!

File: projects/usc-chan/seeds/questionnaires/usc_chan.rb (hunk 1 above)
Comment (emerencia) [reply projects/usc-chan/seeds/questionnaires/usc_chan.rb:81]: You are correct. I don't know why I added another field like this. I'll remove it.

Comment (emerencia) [reply projects/usc-chan/seeds/questionnaires/usc_chan.rb:81]: Fixed.

File: projects/usc-chan/seeds/seeds.rb (hunk 2 above)
Comment (emerencia) [reply projects/usc-chan/seeds/seeds.rb:38]: Oh, nice catch! I'll update it.

Comment (emerencia) [reply projects/usc-chan/seeds/seeds.rb]: Fixed.

File: projects/usc-chan/seeds/seeds.rb (hunk 3 above)
Comment (emerencia) [reply projects/usc-chan/seeds/seeds.rb]: Fixed!

**Pull request merged**